apps/web/src/design-system/icons/index.ts

```diff
@@ -1,3 +1,4 @@
+export { ActiveLabel } from './general/ActiveLabel';
```

Comment:
added missing icons, so they are visible in the storybook

Reply:
Good addition, shame that storybook is down at the moment 😅

Reply:
yeah, we might revisit this question when Nikolay will prepare the new components library designs